code/modules/assembly/flash.dm

```diff
@@ -90,7 +90,7 @@
 		if(M.flash_act(1, 1))
 			M.confused += power
 			terrible_conversion_proc(M, user)
-			M.Stun(1)
+			M.Weaken(2)

```

Comment:
If you make it stun too long then revheads and revs who are recently flashed are fugged.